test/unit/helpers/dom/PrimaryAuthForm.js

```diff
@@ -220,6 +220,42 @@ export default Form.extend({
     return this.socialAuthButton('microsoft');
   },
 
+  githubButton: function() {
+    return this.socialAuthButton('github');
+  },
+
+  gitlabButton: function() {
+    return this.socialAuthButton('gitlab');
+  },
+
+  yahooButton: function() {
+    return this.socialAuthButton('yahoo');
+  },
+
+  lineButton: function() {
+    return this.socialAuthButton('line');
+  },
+
+  paypalButton: function() {
+    return this.socialAuthButton('paypal');
+  },
+
+  paypalSandboxButton: function() {
+    return this.socialAuthButton('paypal_sandbox');
+  },
+
+  salesforceButton: function() {
+    return this.socialAuthButton('salesforce');
+  },
+
+  amazonButton: function() {
+    return this.socialAuthButton('amazon');
+  },
+
+  yahooJapanButton: function() {
```

Comment:
Are these types unused purposefully?

Let's either add tests for them or remove them from our test helpers to avoid unused functions.

Reply:
Not used—was just following along with the PR to add the `APPLE` IdP. Removed in d043e00